eng/SharedFramework.External.props

```diff
@@ -54,7 +54,6 @@
     <!-- Dependencies from dotnet/corefx -->
     <ExternalAspNetCoreAppReference Include="System.IO.Pipelines"                                          Version="$(SystemIOPipelinesPackageVersion)" />
     <ExternalAspNetCoreAppReference Include="System.Security.Cryptography.Xml"                             Version="$(SystemSecurityCryptographyXmlPackageVersion)" />
-    <ExternalAspNetCoreAppReference Include="System.Threading.Channels"                                    Version="$(SystemThreadingChannelsPackageVersion)" />
```

Comment:
@davidfowl We have the new dependencies with channels in the .net shared fx now

Reply:
See https://github.com/aspnet/AspNetCore/issues/10938

Reply:
Yep